radio/src/gui/gui_common.h

```diff
@@ -184,6 +184,26 @@ inline uint8_t MODULE_CHANNELS_ROWS(int moduleIdx)
   }
 }
 
+inline bool isRacingModeAllowed()
+{
+  if (isModulePXX2(INTERNAL_MODULE) && 0 == g_model.moduleData[INTERNAL_MODULE].channelsCount && moduleState[INTERNAL_MODULE].racingMode)
+    return true;
+  else
+    return false;
+}
+
+inline uint8_t IF_ALLOW_RACING_MODE(int moduleIdx)
+{
+  if (!IS_MODULE_ENABLED(moduleIdx)) {
+    return HIDDEN_ROW;
+  }
+  else if (isModulePXX2(moduleIdx)) {
+    if (0 == g_model.moduleData[moduleIdx].channelsCount)
+      return 1;
```

Comment:
return 1 means 2 fields, do we have 2 fields or only one? in this case it should be 0